src/darker/black_diff.py

```diff
@@ -32,11 +32,11 @@
   based on whether reformats touch user-edited lines
 
 """
+
```

Comment:
This formatting changes most probably has nothing to do with dropping Python 3.7 support either. @hugovk was this suggested by a recent Black version, or is there some other reason?

I'd prefer to collect formatting updates in a separate PR.

Reply:
Yes, from Black.